### PR TITLE
Update docs to point to wiki

### PR DESCRIPTION
**Pull request by justinemarie**
:-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,43 +5,22 @@
...
+1. [BESS Overview] (https://github.com/NetSys/bess/wiki/BESS-Overview)
+2. [Build and Install BESS] (https://github.com/NetSys/bess/wiki/Build-and-Install-BESS)
+3. [Write a BESS Configuration Script] (https://github.com/NetSys/bess/wiki/Writing-a-BESS-Configuration-Script)
+4. [Connect BESS to a Network Interface, VM, or Container](https://github.com/NetSys/bess/wiki/Hooking-up-BESS-Ports)
 
-```
-$ git clone https://github.com/NetSys/bess.git
-$ bess/build.py
-```
+To configure and install BESS on Linux quickly, you can run the provided ansible script (ansible-playbook.yml):
```
Comment (changlan) [inline README.md]: ansible-playbook.yml -> `vagrant/bess.yml`

File: README.md [hunk 2]
```diff
@@ -5,43 +5,22 @@
...
-```
-
-Type `run samples/<tab>` on the CLI prompt to run configuration examples. Corresponding files can be found in `bessctl/conf/samples/`.
+    git clone https://github.com/NetSys/bess.git
+    cd bess/
+    sudo apt-get install -y software-properties-common
+    sudo apt-add-repository -y ppa:ansible/ansible
+    sudo apt-get update
+    sudo apt-get install -y ansible
+    ansible-playbook ansible-playbook.yml
```
Comment (changlan) [inline README.md]: ansible-playbook.yml -> vagrant/bess.yml

**Comment by justinemarie** [comment]
Fixed! Thanks, Chang!

**Comment by justinemarie** [comment]
Chang, could you merge?

**Comment by changlan** [comment]
Waiting for the CI builds. Will merge once it is finished :)

**Comment by changlan** [comment]
image

**Pull request merged**